config/initializers/wrap_parameters.rb

```diff
@@ -7,7 +7,7 @@
   wrap_parameters :format => [:json]
 end
 
-# Enable root element in JSON by default.
+# Disable root element in JSON by default.
 ActiveSupport.on_load(:active_record) do
-  self.include_root_in_json = true
+  self.include_root_in_json = false
```

Comment:
This is the default in rails 6:
https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/serialization.rb#L10

So i think we can just drop it from the initializer.